app/assets/stylesheets/blog.css.scss

```diff
@@ -0,0 +1,10 @@
+//
+// BLOG PAGE
+//
+
+
+.blog-body {
+  margin-top: 6em;
```

Comment:
Can you set this to: 'margin-top: 125px;' That way this is the same margin than we use for the resources page and it looks consistent. 

I am curious to see it with actual blog posts! :)